e2e/assets/single-cluster/helm-kustomize.yaml

```diff
@@ -0,0 +1,8 @@
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: helm-kustomize
```

Comment:
namespace needs to be provided, i.e. `fleet-default`

Reply:
should we also change the name here to be `single-cluster-helm-kustomize`? Same comment for the other GitRepo

Reply:
If we change the name, we should also change it in fleet-examples.